autotest/gdrivers/tiledb_write.py

```diff
@@ -40,34 +40,44 @@ def test_tiledb_write_complex():
 
     src_ds = gdal.Open('../gcore/data/cfloat64.tif')
 
-    new_ds = gdaltest.tiledb_drv.CreateCopy('tmp/tiledb_complex64', src_ds)
+    for mode in [ 'BAND', 'PIXEL', 'ATTRIBUTES' ]:
```

Comment:
a more pytest'ic way would be to use pytest.parametrize (see autotest/gdrivers/fits.py for an example)